### PR TITLE
[RETINA-313] small bugs

### PR DESCRIPTION
**Pull request by Jbcampbe**

### CODE REVIEW

File: src/routes/tool-detail.vue [hunk 1]
```diff
@@ -919,16 +919,87 @@ export default {
...
               variables: {
                 tool_id: this.getTool.id,
                 decomissioned_status: newStatus,
                 decomission_reason: result.value
-              }
+              },
+              refetchQueries: [
+                {
+                  query: gql`query tools($pagingParameters: PagingParameters) {
+                    searchTool(pagingParameters: $pagingParameters) {
```
Comment (ekraihan) [inline src/routes/tool-detail.vue]: Searching on tool detail?

File: src/routes/tool-detail.vue [hunk 2]
```diff
@@ -919,16 +919,87 @@ export default {
...
+                  variables: {
+                    pagingParameters: {
+                      page_size: 15,
+                      page_number: 0
+                    }
+                  }
+                },
+                {
+                  query: gql`query selectedTools($tool_ids: [ID!]!) {
+                    getMultipleTool(tool_ids: $tool_ids) {
```
Comment (ekraihan) [inline src/routes/tool-detail.vue]: Muiltiple tool on tool detail?

File: src/routes/tool-detail.vue (hunk 1 above)
Comment (Jbcampbe) [reply src/routes/tool-detail.vue]: refetching the queries to update the cache

File: src/routes/tool-detail.vue (hunk 2 above)
Comment (Jbcampbe) [reply src/routes/tool-detail.vue]: refetching the queries to update the cache

Comment (Jbcampbe) [reply src/routes/tool-detail.vue]: although actually I don't think I have to have that one

File: src/routes/tool-detail.vue [hunk 3]
```diff
@@ -919,16 +919,55 @@ export default {
...
+                  variables: {
+                    pagingParameters: {
+                      page_size: 15,
+                      page_number: 0
+                    }
+                  }
+                }
+              ]
             }).then(() => {
+              this.$store.commit('setToolSelection', this.getTool.id, false)
```
Comment (ekraihan) [inline src/routes/tool-detail.vue]: Some (probably ignorant) questions
- How do you know that the search will pull down the id of the tool that you decomissioned?
- If you aren't actually looking for that specific tool, could you have the paging parameters be 0 and 0?
- If you can just do 0 and 0, maybe there is a better way to invalidated tool that you decomissioned in the cache?

Comment (Jbcampbe) [reply src/routes/tool-detail.vue]: 1. We are on the detail page for the tool you are decommissioning so I already have the id
2. I don't think that would successfully update the cache : /

**Pull request merged**